HighTeach/urls.py

```diff
@@ -19,5 +19,6 @@
 
 urlpatterns = [
     path('admin/', admin.site.urls),
-    path('', views.homePage, name='homePage')
+    path('', views.homePage, name='homePage'),
+    path('logout/', views.CustomLogoutView.as_view(), name='logout'),
```

Comment:
Why do you need to create a custom view? You can do something like this:
```python
from django.contrib.auth import views as auth_views
urlpatterns = [
    path('logout/', auth_views.LogoutView.as_view(template_name='users/logout.html'), name='logout'),
]
```

Reply:
You are right, I changed it.